packages/components/app/styles/@hashicorp/design-system-power-select-overrides.scss

```diff
@@ -71,13 +71,24 @@
 
   .ember-basic-select-dropdown,
   .ember-power-select-dropdown {
+    &.ember-basic-dropdown-content--below,
     &.ember-basic-dropdown-content--in-place {
       margin-top: -3px;
       border: var(--token-form-control-border-width) solid var(--token-form-control-base-border-color-default);
       border-bottom-right-radius: 6px;
       border-bottom-left-radius: 6px;
       box-shadow: var(--token-elevation-high-box-shadow), var(--token-elevation-high-box-shadow);
     }
+
+    &.ember-basic-dropdown-content--above {
+      margin-top: 3px;
+      border: var(--token-form-control-border-width) solid var(--token-form-control-base-border-color-default);
+      border-top-left-radius: 6px;
+      border-top-right-radius: 6px;
+      border-bottom-right-radius: 0;
+      border-bottom-left-radius: 0;
```

Comment:
You wouldn't want to combine the radius styles?

`border-radius: 6px 6px 0 0;`

Reply:
It was a lazy copy-past, but yes we should contract these – updates as such